plugin/log/setup.go

```diff
@@ -62,7 +62,7 @@ func logParse(c *caddy.Controller) ([]Rule, error) {
 			})
 		} else {
 			// Name scope, and maybe a format specified
-			format := DefaultLogFormat
+			var format string
```

Comment:
are these things really an improvement?

Reply:
@miekg I guess it is something "nice to have" instead of "true" improvement.

Though the reason `format := DefaultLogFormat` shown up as a warning is that down in Ln 72:
```
 			default:
 				format = args[1]
```
That means `DefaultLogFormat` is actually not in use.